jenkins-pipelines/artifacts-amazon2.jenkinsfile

```diff
@@ -7,7 +7,6 @@ artifactsPipeline(
     test_config: 'test-cases/artifacts/amazon2.yaml',
     backend: 'aws',
     region_name: 'eu-west-1',
-    provision_type: 'spot_low_price',
```

Comment:
Why do you remove theses from the job?

Reply:
since it's the wrong default, after changes @juliayakovlev related to the spot,
as @bentsi "spotted" here
https://github.com/scylladb/scylla-cluster-tests/pull/2915#discussion_r528390867

Reply:
the artifact should pipeline has it spot as default, no point of have it on all pipeline again.

Reply:
In this case it worth to leave it as spot_low_price, these are very short tests but running multiple times.
The spot duration is more expensive and the chance to get spot termination in 15 mins test is very low.

Reply:
o.k. i'll remove this part. and leave as is